dapps/evancore.vue.libs/src/component.ts

```diff
@@ -69,11 +70,17 @@ export default class EvanComponent extends Vue {
    */
   testMode = false;
 
+  /**
+   * Current dapp's dispatcher handler, so it can be accessed easier
+   */
+  dispatcher: EvanVueDispatcherHandler = null;
+
   constructor() {
     super();
 
     this.dapp = this.activeDApp();
     this.domainName = getDomainName();
+    this.dispatcher = this.$store.state.dispatcher;
```

Comment:
isn't there a high chance, that we confuse the vuex dispatcher with the evan-dispatcher?

Reply:
true true true, i removed it